bmi_lstm.py

```diff
@@ -640,7 +684,6 @@ def get_grid_shape(self, grid_id, shape):
     #------------------------------------------------------------ 
     def get_grid_size(self, grid_id):
        
-        # JG Edit
         # 0 is the only id we have
         if grid_id == 0:
             return 1
```

Comment:
Does BMI Python also have the option to return `BMI_FAILURE` like Fortran/C?

Reply:
Not exactly, but more so raising some flavor of a `NotImplemented` function...  
```
def get_grid_edge_count(self, grid):
        raise NotImplementedError("get_grid_edge_count")
```
See the first bullet in [best practices doc](https://bmi.readthedocs.io/en/latest/bmi.best_practices.html#best-practices)